src/pybind/mgr/ansible/ansible_runner_svc.py

```diff
@@ -8,7 +8,7 @@
 import collections
 import logging
 try:
-    from typing import Optional
+    from typing import Optional, Dict, Any, List, Set
```

Comment:
could you move these mypy related changes into another commit?

Reply:
That would be artificial. I did all of those fixes in one go:

1. remove `mgr/ansible/tox.ini` + cmake integration
2. add ansible to `mgr/tox.ini`
3. Fix all issues that did pop up when doing so.

Reply:
i believe atomic changes would help reviewers to understand them better.

Reply:
Actually this is one big atomic change. Either you don't check ansible with mypy, then you don't need to change anything. Or, you do check ansible with mypy and then you'll need to make all of these changes.